components/TextFields/src/Theming/MDCTextInputControllerUnderline+MaterialTheming.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialContainerScheme.h"
+#import "MaterialTextFields.h"
+
+/**
+ This category is used to style MDCTextInputControllerUnderline instances to a specific Material
+ style which can be found within the [Material
+ Guidelines](https://material.io/design/components/text-fields.html).
+ */
+@interface MDCTextInputControllerUnderline (MaterialTheming)
```

Comment:
It would be good to have a visual example of this theming as part of this PR. Would you please either add Snapshot tests for this extension or else grab a screenshot of it from an Example?

Reply:
I've added two screenshots.